cocotb/__init__.py

```diff
@@ -31,6 +31,20 @@
 See https://docs.cocotb.org for full documentation
 """
 import os
+
+_library_coverage = None
+if "COCOTB_LIBRARY_COVERAGE" in os.environ:
+    try:
+        import coverage
+    except ImportError:
+        raise
+    else:
+        _library_coverage = coverage.coverage(
+            data_file=".library_coverage",
```

Comment:
This might not match the default patterns codecov looks for.

Reply:
Does codecov search the tree for `.coverage` files? I was planning on using `coverage combine`.

Looks like `codecov` supports the `-f` option, so the name doesn't matter and we can skip merging.